website/components/ApiMeta.tsx

```diff
@@ -20,25 +19,45 @@ export interface ApiMetaProps {
 }
 
 export function ApiMeta(props: ApiMetaProps) {
-  const lang = useLang();
-  const href = `/${lang}/misc/planning/future`;
   const tagStyle = props.inline ? styles.tagInline : styles.tag;
   const wrapperStyle = props.inline ? styles.wrapperInline : styles.wrapper;
+
+  const getGitTagHref = (version: string) =>
+    `https://github.com/web-infra-dev/rspack/releases/tag/v${version.replace('v', '')}`;
+
   return (
     <div className={wrapperStyle}>
       {props.addedVersion && (
         <span className={`${tagStyle} ${styles.added}`}>
-          <a href={href}>Added in v{props.addedVersion}</a>
+          <a
+            href={getGitTagHref(props.addedVersion)}
+            target="_blank"
+            rel="noreferrer"
+          >
+            Added in v{props.addedVersion}
+          </a>
         </span>
       )}
       {props.deprecatedVersion && (
         <span className={`${tagStyle} ${styles.deprecated}`}>
-          <a href={href}>Deprecated in v{props.deprecatedVersion}</a>
+          <a
+            href={getGitTagHref(props.deprecatedVersion)}
```

Comment:
How about linking deprecated version to https://rspack.dev/misc/planning/future#deprecation?